pages/common/sort.md

```diff
@@ -19,9 +19,9 @@
 
 `sort -n {{path/to/file}}`
 
-- Sort the passwd file by the 3rd field, numerically:
+- Sort a file, printing the output to the specified output file (can be used to sort a file in-place):
 
-`sort -t: -k 3n /etc/passwd`
+`sort --output={{path/to/file}} {{path/to/file}}`
```

Comment:
I'm not sure it is ok to replace the origin case, in my opinion, sort by field is used often.